Code/Mantid/Framework/PythonInterface/plugins/algorithms/ApplyNegMuCorrection.py

```diff
@@ -0,0 +1,94 @@
+from mantid.api import *  # PythonAlgorithm, registerAlgorithm, WorkspaceProperty
+
+#pylint: disable=no-init, too-many-arguments
+class ApplyNegMuCorrection(PythonAlgorithm):
+
+    #Combining work spaces and normalising the correction.
+    #_OILE and _OIHE are unused parameters for now to make PyLint happy
+    def combine(self,dataDir,runno,A2000,B2000,A3000,B3000,_OILE,_OIHE,spec):
+        if spec<10:
+            specNo='0'+str(spec)
+        else:
+            specNo=str(spec)
+        print dataDir+'ral0'+runno+'.rooth30'+specNo+'.dat'
+        #loading data
+        try:
+            ws3000=Load(Filename=dataDir+r'\ral0'+runno+'.rooth30'+specNo+'.dat', OutputWorkspace='ws3000')
+            print 'hello'
+            ws2000=Load(Filename=dataDir+r'\ral0'+runno+'.rooth20'+specNo+'.dat', OutputWorkspace='ws2000')
+        except RuntimeError:
+            print runno+' '+specNo+"not found"
```

Comment:
I'd suggest adding amore informative message in this except.
Also, should it actually raise the exception? I understand that if this `Load()` fails the algorithm won't run correctly.